great_expectations/experimental/datasources/spark_datasource.py

```diff
@@ -119,17 +120,21 @@ def add_csv_asset(
         Args:
             name: The name of the csv asset
             regex: regex pattern that matches csv filenames that is used to label the batches
+            glob_directive (str): glob for selecting files in directory (defaults to `**/*`) or nested directories (e.g. `*/*/*.csv`)
```

Comment:
Same comment

Reply:
Good to know that we discover this automatically using the annotations!  Makes perfect sense; will update shortly.  Thank you!

Reply:
All incorporated -- thanks @NathanFarmer !